armi/physics/neutronics/tests/test_crossSectionManager.py

```diff
@@ -172,9 +230,8 @@ def test_createRepresentativeBlock(self):
 
     def test_createRepresentativeBlockDissimilar(self):
         """
-        Test creation of a representative block from a collection with dissimilar blocks
+        Test creation of a representative block from a collection with dissimilar blocks/
```

Comment:
```suggestion
        Test creation of a representative block from a collection with dissimilar blocks
```

Does ruff not want this all on one line?